### PR TITLE
Enable system prune test remote client

### PR DESCRIPTION
**Pull request by sujil02**
Removed the use of lastest flags as not supported in remote client
and latest flag alredy teseted in indivisual pod start stop test.

Signed-off-by: Sujil02 <sushah@redhat.com>

### CODE REVIEW

File: test/e2e/prune_test.go [hunk 1]
```diff
@@ -320,9 +322,6 @@ var _ = Describe("Podman prune", func() {
 		create.WaitWithDefaultTimeout()
 		Expect(create.ExitCode()).To(Equal(0))
 
-		// Adding images should be pruned
-		podmanTest.BuildImage(pruneImage, "alpine_bash:latest", "true")
```
Comment (rhatdan) [inline test/e2e/prune_test.go:324]: Why are you removing this?

Comment (sujil02) [reply test/e2e/prune_test.go:324]: Build image with layers is not implemented by a remote client. And for this use case it simply added a bunch of more images using build image and checks if they are pruned with system prune command. However the addition of BB image here https://github.com/containers/libpod/pull/6318/files#diff-9e3f37b4287d658db5658028b861b77bL319 and validating it pruned here https://github.com/containers/libpod/pull/6318/files#diff-9e3f37b4287d658db5658028b861b77bL351
addresses the use case hence the test need not be skipped for that matter and is still accounts as a valid test.

**Comment by rhatdan** [comment]
LGTM
Although I would like to get podman-remote build to work.

**Pull request merged**